wc3-hw1-TranNgocLam/cypress/pages/booksPage.js

```diff
@@ -0,0 +1,28 @@
+export const booksPage = {
+    LBL_USERNAME_VALUE: "#userName-value",
+    BTN_OK: "#closeSmallModal-ok",
+    TB_BOOK: '.ReactTable a',
+
+    addBookToCollection(book) {
+        cy.get(this.TB_BOOK).contains(book).click();
+        cy.get('button').contains('Add To Your Collection').click({ force: true });
+        cy.on("window:confirm", (s) => {
+            const runout = ['Book added to your collection', 'Book already present in your collection!']
```

Comment:
This will make you pass the test even when book is already in collection, and you cannot make sure that book is just added or exist in collection when add